website/templates/emails/discussions_email_rejected.txt.mako

```diff
@@ -0,0 +1,19 @@
+Hello${' {}'.format(user.fullname) if user else ''},
+
+The email that you sent to ${target_address} was rejected, and was therefore not forwarded.
+
+%if reason == 'no_user':
```

Comment:
Visual inspection suggests that we have the strings right here. Do we have tests for these?